runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightOverlay.java

```diff
@@ -85,13 +85,20 @@ public Dimension render(Graphics2D graphics)
 			case "Walk here":
 			case "Cancel":
 			case "Continue":
+			case "Scroll map":
+			case "Focus":
 				return null;
 			case "Move":
 				// Hide overlay on sliding puzzle boxes
 				if (target.contains("Sliding piece"))
 				{
 					return null;
 				}
+			case "Open":
+				if (target.equalsIgnoreCase("map"))
```

Comment:
`"map".equalsIgnoreCase(target)`
always better to do like this, so there is no possibility of `NullPointerException`